lib/routes/veeam/head.js

```diff
@@ -35,7 +35,7 @@ function headVeeamFile(request, response, bucketMd, log) {
             headless: true,
         });
         const dataBuffer = Buffer.from(buildHeadXML(builder.buildObject(fileToBuild)));
-        return responseContentHeaders(null, {}, getResponseHeader(request, request.bucketName,
+        return responseContentHeaders(null, {}, getResponseHeader(request, data,
```

Comment:
Worth adding a test if possible to ensure that the CORS headers are returned, usually we ensure that each bugfix ticket is merged with a test to ensure that we never recreate the issue again